docs/development/development_guide_en.md

```diff
@@ -0,0 +1,10 @@
+# Guide to development
+
+## Local Deploy
+
+1. the environment variable KUBECONFIG must have a valid config for the target cluster.
+1. execute `make run` to run the service discovery operator locally
```

Comment:
Add a hint that you have to set the WATCH_NAMESPACE environment variable if you are running your CES inside a namespace